### DISCUSSION TITLE
Linuxbrew: gcc conflicting with gcc@5 during Xorg installation

### DISCUSSION THREAD

**Discussion by mikesep** [Homebrew/discussions#829] [Linux] [answered]
I've noticed today that `gcc` and `gcc@5` can't be installed at the same time.

```
/home/linuxbrew/.linuxbrew/bin/c++-5 -> /home/linuxbrew/.linuxbrew/Cellar/gcc@5/5.5.0_6/bin/c++-5
/home/linuxbrew/.linuxbrew/bin/cpp-5 -> /home/linuxbrew/.linuxbrew/Cellar/gcc@5/5.5.0_6/bin/cpp-5
/home/linuxbrew/.linuxbrew/bin/g++-5 -> /home/linuxbrew/.linuxbrew/Cellar/gcc@5/5.5.0_6/bin/g++-5
/home/linuxbrew/.linuxbrew/bin/gcc-5 -> /home/linuxbrew/.linuxbrew/Cellar/gcc@5/5.5.0_6/bin/gcc-5
```

I ran into this when installing packages on a system with an antiquated gcc version. I think core `brew` installs `gcc@5` as of [this commit](https://github.com/Homebrew/brew/commit/221983dbcff442fa98d28c36cb2fb74af376d937).

I tried to install `graphviz`, but it depends on many things from [homebrew-xorg](https://github.com/Linuxbrew/homebrew-xorg) including `xrdb`, which [depends on gcc](https://github.com/Linuxbrew/homebrew-xorg/blob/7eb784e5a56d3e6ec5f4f63b5456657431dbb97d/Formula/xrdb.rb#L18).

- Is it intentional that `gcc` and `gcc@5` conflict?
- Should `xrdb` depend on a pinned gcc version like `gcc@5`?

-----

Meta: I was going to report this as an Issue in linuxbrew-core, but the [Issue chooser UI](https://github.com/Homebrew/linuxbrew-core/issues/new/choose) doesn't seem to have an option for that -- the last link just points to the same page. Is that intended?

**Comment by gromgit** [comment, marked as answer]
Please see #809 for a related discussion and suggested resolution.

**Reply by mikesep** [reply]
Thanks @gromgit! I had checked for Issues but didn't see that previous discussion thread.
